org.mixedrealitytoolkit.input/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 ### Fixed
 
 * Add logic to account for a bound but untracked interaction profile. [PR #704](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/704)
+* Ensures the simulated input sources hold their state (including gestures) when their toggle state is locked on. [PR #705](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/705)
 
 ## [3.2.0-development] - 2024-03-20
```

Comment:
@keveleigh ...please remove the old "-development"  .   Only the top version should have -development.

I really need to have some sort of automated process for tagging the CHANGELOG versions

Reply:
Removed! It'd also be nice to update the existing automated checks to support "Unreleased" instead of a date next to the version (or, I guess [KeepAChangelog just uses "Unreleased" _as_ the version](https://keepachangelog.com/en/1.1.0/), but either way) so we don't have to put some random date or get flagged in the PR validation pipeline.

Reply:
Actually, that might have been from the Unity's package validation package. Bummer :(

Reply:
Looks good now.